packages/core/ios/EmbraceManager.swift

```diff
@@ -21,83 +22,94 @@ private let EVENT_ATTRIBUTES_KEY = "attributes"
+    public let disableNetworkSpanForwarding: Bool
+
+    public init(from: NSDictionary) {
+        self.appId = from["appId"] as? String ?? ""
+        self.appGroupId = from["appGroupId"] as? String
+        self.disableCrashReporter = from["disableCrashReporter"] as? Bool ?? false
+        self.disableAutomaticViewCapture = from["disableAutomaticViewCapture"] as? Bool ?? false
+        self.endpointBaseUrl = from["endpointBaseUrl"] as? String
+        self.disableNetworkSpanForwarding = from["disableNetworkSpanForwarding"] as? Bool ?? false
+    }
 }
 
 @objc(EmbraceManager)
 class EmbraceManager: NSObject {
-  private var log = OSLog(subsystem: "Embrace", category: "ReactNativeEmbraceManager")
-  private var spanRepository = SpanRepository()
+    private var log = OSLog(subsystem: "Embrace", category: "ReactNativeEmbraceManager")
+    private var spanRepository = SpanRepository()
+    private var config: SDKConfig = SDKConfig(from: NSDictionary())
 
-  @objc(setJavaScriptBundlePath:resolver:rejecter:)
-  func setJavaScriptBundlePath(_ path: String, resolver resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) {
-    DispatchQueue.global(qos: .background).async {
-        do {
-            let bundleID = try computeBundleID(path: path)
-            try Embrace.client?.metadata.addResource(key: REACT_NATIVE_BUNDLE_ID_RESOURCE_KEY, value: bundleID.id, lifespan: .process)
-            resolve(true)
-        } catch let error {
-          reject("SET_JS_BUNDLE_PATH_ERROR", "Error setting JavaScript bundle path", error)
+    @objc(setJavaScriptBundlePath:resolver:rejecter:)
+    func setJavaScriptBundlePath(_ path: String, resolver resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) {
+        DispatchQueue.global(qos: .background).async {
+            do {
+                let bundleID = try computeBundleID(path: path)
+                try Embrace.client?.metadata.addResource(key: REACT_NATIVE_BUNDLE_ID_RESOURCE_KEY, value: bundleID.id, lifespan: .process)
+                resolve(true)
+            } catch let error {
+                reject("SET_JS_BUNDLE_PATH_ERROR", "Error setting JavaScript bundle path", error)
+            }
         }
     }
-  }
+
     @objc
     func getDefaultJavaScriptBundlePath(_ resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) {
-        if  let filePath = Bundle.main.path(forResource: "main", ofType: "jsbundle"){
+        if  let filePath = Bundle.main.path(forResource: "main", ofType: "jsbundle") {
             resolve(filePath)
         } else {
             reject("error", "Unable to retrieve JS bundle path", nil)
         }
     }
+
     @objc
     func isStarted(_ resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) {
         if let embraceStarted = Embrace.client?.started {
-        resolve(embraceStarted)
+            resolve(embraceStarted)
```

Comment:
you can probably close https://www.notion.so/embraceio/Re-run-swiftlint-over-the-repo-1287e3c99852802390a4ed824e537726?pvs=4 once this is merged as well as this reformatting was some existing debt